sympy/simplify/sqrtdenest.py

```diff
@@ -236,6 +236,18 @@ def _sqrtdenest0(expr):
         else:
             n, d = [_sqrtdenest0(i) for i in (n, d)]
             return n/d
+
+    if isinstance(expr, Add):
+        cs = []
+        args = []
+        for arg in ordered(expr.args):
```

Comment:
It seems that `ordered` puts the small square roots first. I have no idea if that would be favourable for denesting or not but, I'm willing to accept your word. Perhaps there could be an explanation in a comment.